vgen/src/main/resources/io/gapi/vgen/java/java_gapic.yaml

```diff
@@ -1,8 +1,10 @@
 type: io.gapi.vgen.ConfigProto
 language: java
-language_provider: io.gapi.vgen.java.JavaGapicLanguageProvider
-snippet_files:
-- main.snip
-- settings.snip
-fragment_files:
-- fragment.snip
+templates:
+- language_provider: io.gapi.vgen.java.JavaGapicCodeLanguageProvider
+  snippet_files:
+  - main.snip
+  - settings.snip
+- language_provider: io.gapi.vgen.java.JavaGapicFragmentLanguageProvider
+  snippet_files:
+  - fragment.snip
```

Comment:
@bjwatson , by including the fragment file in GAPIC generation, you are changing the behavior, and in an undesirable way (from the perspective of the Java codegen). I'd much prefer the behavior to stay the same with this refactoring.